### PR TITLE
Add routing

### PR DESCRIPTION
**Pull request by rockycodes**
I built this on top of Sarah's branch so her changes are in here too. Sorry it's kinda mess merp. But it's working!!!

The biggest change is that `getDirections` function in `HomeScreen.js`. It gets the users location and uses the Google Directions api to fetch the route from there to the selected landmark. Then it decodes the response using this crazy function I found here: https://github.com/react-native-community/react-native-maps/issues/929 and sets the coordinates on state.

I added Polyline component to the Maps component which the coordinates get fed into.

Todo:
- A way to clear directions
- Force the drawer to close when the user clicks "Directions"
- Error handling for if directions can't be fetched

### CODE REVIEW

**Comment by bramus** [comment]
You could also use a pre-built standalone component for this https://github.com/bramus/react-native-maps-directions ;)

File: screens/HomeScreen.js [hunk 1]
```diff
@@ -78,6 +82,64 @@ export default class HomeScreen extends React.Component {
     });
   }
 
+  getDirections = async landmarkCoordinates => {
+    const location = await Location.getCurrentPositionAsync({});
+
+    // is there a cuter es6 way to assign these variables?
+    const initialLat = location.coords.latitude;
+    const initialLong = location.coords.longitude;
```
Comment (a-Daley) [inline screens/HomeScreen.js]:
```suggestion
    const { coords: { latitude: initialLat, longitude: initialLong } } = location;
```

File: screens/HomeScreen.js [hunk 2]
```diff
@@ -78,6 +82,64 @@ export default class HomeScreen extends React.Component {
     });
   }
 
+  getDirections = async landmarkCoordinates => {
+    const location = await Location.getCurrentPositionAsync({});
+
+    // is there a cuter es6 way to assign these variables?
+    const initialLat = location.coords.latitude;
```
Comment (a-Daley) [inline screens/HomeScreen.js]:
```suggestion
    
```

File: screens/HomeScreen.js [hunk 3]
```diff
@@ -78,6 +82,64 @@ export default class HomeScreen extends React.Component {
...
+    const initialLong = location.coords.longitude;
+    const finalLat = landmarkCoordinates.latitude;
+    const finalLong = landmarkCoordinates.longitude;
```
Comment (a-Daley) [inline screens/HomeScreen.js]:
```suggestion
```

File: screens/HomeScreen.js [hunk 4]
```diff
@@ -78,6 +82,64 @@ export default class HomeScreen extends React.Component {
     });
   }
 
+  getDirections = async landmarkCoordinates => {
+    const location = await Location.getCurrentPositionAsync({});
+
+    // is there a cuter es6 way to assign these variables?
+    const initialLat = location.coords.latitude;
+    const initialLong = location.coords.longitude;
+    const finalLat = landmarkCoordinates.latitude;
```
Comment (a-Daley) [inline screens/HomeScreen.js]:
```suggestion
    const { latitude: finalLat, longitude: finalLong } = landmarkCoordinates;
```

**Pull request merged**